blazor/dropdown-list/value-binding.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: Value Binding in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Value Binding in Syncfusion Blazor DropDown List component and more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Value Binding in Dropdown List
+
+The value binding as the process of passing values between a component and its parent. There are two methods for binding values, as shown below.

```

Comment:
Value binding is the process of passing values between a component and its parent. There are two methods for binding values. These are: